server.js

```diff
@@ -3,36 +3,63 @@ import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
+import goldenGlobesData from './data/golden-globes.json'
 
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
+const port = process.env.PORT || 9000
 const app = express()
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
+
+const Globe = new mongoose.model('Globe', {
+    year_film: Number,
+    year_award: Number,
+    ceremony: Number,
+    category: String,
+    nominee: String,
+    film: String,
+    win: {
+      type: Boolean,
+      required: true 
+    } 
+});
+
+
+if (process.env.RESET_DATABASE) {
+
+  const populateDatabase = async () => {
+    await Globe.deleteMany();
+
+    goldenGlobesData.forEach(item => {
+      const newGlobe = new Globe(item)
+      newGlobe.save();
+    })
+  }
+  populateDatabase();
+}
+
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Hello Internet')
+})
+
+app.get('/globes', async (req, res) => {
+  const allGlobes = await Globe.find(req.query);
+  res.json(allGlobes);
+})
+
+app.get('/globes/:nominee', async (req, res) => {
```

Comment:
`nominee` is not a good key for extracting single document, unfortunately. When you use it like this you returning a bad set of data because you cutting off a big portion of it. For example "Avatar" has a lot of nominations, but you are showing only the first one.

You need to use field which is always unique for every document, usually, it is `id` field.

Reply:
Changed it to a find so it shows all the nominations an actor or movie have.

Reply:
If you change this one to `find()` it will make your data complete, but you still need to provide the endpoint to the single element by using `_id` as a key

Reply:
That specific data  didn't have an Id key. 
But I tried solving it using multiple paths..